### PR TITLE
feat: alert for vue-showcase

### PR DESCRIPTION
**Pull request by annsch**
- examples and variants for DBAlert Component for Vue Showcase

### CODE REVIEW

File: showcases/vue-showcase/src/components/alert/Alert.vue [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+defaultComponentVariants[1].examples[4].code = `<DBAlert variant="warning" headline="Headline" :link=\"{href: "#"}\">Alert ${linkSlot}</DBAlert>`;
+
+// Content
+defaultComponentVariants[2].examples[0].code = `<DBAlert icon="account" headline="Headline" :link=\"{href: "#"}\">Headline & Text & Link & Icon ${linkSlot}</DBAlert>`;
+defaultComponentVariants[2].examples[1].code = `<DBAlert headline="Headline" :link=\"{href: "#"}\">Headline & Text & Link ${linkSlot}</DBAlert>`;
+defaultComponentVariants[2].examples[2].code = `<DBAlert :link=\"{href: "#"}\">Text & Link ${linkSlot}</DBAlert>`;
+defaultComponentVariants[2].examples[3].code = `<DBAlert>Text</DBAlert>`;
+
+// Behaviour
+defaultComponentVariants[3].examples[0].code = `<DBAlert headline="Headline" :link=\"{href: "#"}\" onClick={()=>{alert("click close button")}}>Alert ${linkSlot}</DBAlert>`;
```
Comment (nmerget) [inline showcases/vue-showcase/src/components/alert/Alert.vue:34]: I guess you would use `@click` here right?

**Comment by mfranzke** [comment]
@annsch will we follow up with this approach or wait for Nicos suggested solution?

**Comment by annsch** [comment]
> @annsch will we follow up with this approach or wait for Nicos suggested solution?
@mfranzke  I'd prefer the last

**Comment by mfranzke** [comment]
I fear that this will change a lot again through https://github.com/db-ui/mono/pull/724

**Comment by mfranzke** [comment]
@annsch could you please adapt the latest changes by Nico to this PR?

**Pull request closed without merging**